specification/src/main/asciidoc/platform/Profiles.adoc

```diff
@@ -147,7 +149,7 @@ annotations defined by the Common Annotations specification ( _Resource_,
  _Resources_, _PostConstruct_, _PreDestroy_ )
 
 The following functionality is required to be
-supported in all Jakarta EE profiles:
+supported in all Jakarta EE profiles other than the Core Profile:
```

Comment:
I think this whole section "Requirements for All Jakarta EE Profiles" needs to be re-worked.  Just inserting the "other than Core Profile" phrase opens the door for other Profiles to also not follow what's listed in this section.  Maybe this section just only contain the Java SE requirements and Common Annotations?  And, the requirement for Transactions and java: naming context should be moved to the Web Profile?  Just an idea.  But, we need to do something more than just inserting the "other than Core Profile" phrase.

Reply:
The simplest would be to remove JNDI/JTA from this section and add it as required in the Web Profile.